python/langsmith/run_helpers.py

```diff
@@ -123,8 +124,11 @@ def tracing_context(
         client: The client to use for logging the run to LangSmith. Defaults to None,
         enabled: Whether tracing is enabled. Defaults to None, meaning it will use the
             current context value or environment variables.
-        replicas: A sequence of tuples containing project names and optional updates for each replica.
-            Example: [("my_experiment", {"reference_example_id": None}), ("my_project", None)]
+        replicas: A sequence of replicas to send runs to. Can be either:
+            - Legacy format: tuples of (project_name, updates)
```

Comment:
Meta note: every single time I use a tuple I feel like I end up really regretting it 😂

Reply:
Though I don't think we formally deprecated the shorthand in JS